Xamarin.MacDev/EntitlementExtensions.cs

```diff
@@ -59,7 +59,24 @@ public static class EntitlementKeys
 		public const string PhotosLibrary = "com.apple.security.personal-information.photos-library";
 		public const string AppleEvents = "com.apple.security.automation.apple-events";
 
-		static string[] allKeys;
+        public const string AppAttest = "com.apple.developer.devicecheck.appattest-environment";
```

Comment:
Indentation is off (we use tabs in this repo).

Reply:
should be fixed now! thanks for pointing it out